apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -382,7 +382,7 @@ def prepare_tag(
         assert_never(source_blocks_order)
 
     # prep for visual pairing
-    correct_tags = {block["tag"] for block in all_blocks if block["tag"] is not None}
```

Comment:
AFAIK block["tag"] should never be None.

Reply:
at this point, ya that should be true because `get_graph_info` above replaces tags of `None` with a random `UUID`.

Reply:
can we encode this fact in the type declarations?

Reply:
I believe it was encoded: it determined that `block["tag"]` would always be a string, and wanted me to remove this check. Unless you are referring to something else?

Reply:
@SethPoulsen please check this.

Reply:
no, that sounds good